.dockerignore

```diff
@@ -0,0 +1,7 @@
+target/
```

Comment:
You already have the dockerignore file, why not doing `COPY ./ ./` then?

Reply:
Will give it another try but it caused problems with poisoned caches :(

Reply:
Actually, I'd categorize this as an optimisation and do it later. Fine for you?

Reply:
Yes it's just a risk because if we move the crates around we may notice that we forgot to update the dockerfile very late in the process (at release time)